test/e2e/testutils/util/get_test_variable.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019-2021 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	testutils "github.com/liqotech/liqo/pkg/utils"
+	cachedclientutils "github.com/liqotech/liqo/pkg/utils/cachedClient"
+)
+
+// GetEnvironmentVariable retrieves the value of the environment variable named by the key.
+// If the variable is not present calls klog.Fatal().
+func GetEnvironmentVariable(key string) string {
```

Comment:
```suggestion
func GetEnvironmentVariableOrDie(key string) string {
```

Nit. Just to explicit the behavior